genesis.md

```diff
@@ -481,3 +481,43 @@ same principle can be used to scale up other aspects of
 deployments, such as CPU, memory, or disk sizes — just keep in
 mind this may need to be done indirectly by specifying instance or
 disk types and adding the desired resources in your cloud-config.
+
+## Stemcell Version Upgrades
+
+Genesis provides a simple way for you to update stemcells for deployments based off of Genesis Kits.  The typical process is to upload a new stemcell to the BOSH Director and perform a `genesis deploy` on each deployment.
+
+In each kit there is a default for the os flavor and version:
+
+```
+params:
+  stemcell_os:      ubuntu-trusty
+  stemcell_version: 3468.latest
+```
+
+To upgrade the stemcell version for a deployment start by uploading a new stemcell to the BOSH Director.  Full instuctions for doing this are located in the [BOSH Runbook under Uploading a Stemcell](https://runbooks.starkandwayne.com/bosh.html#upload-a-stemcell).  It should be noted that most kits ship pre-compiled BOSH releases and do not allow the major version of OS of the stemcell to be changed.  If you are currently using `3468.22` you should upload a newer version in the `3468` series, such as `3468.23`.
```

Comment:
Using a path-only URL allows us to rename runbooks.starkandwayne.com (should we choose to do so).

```suggestion
To upgrade the stemcell version for a deployment start by uploading a new stemcell to the BOSH Director.  Full instuctions for doing this are located in the [BOSH Runbook under Uploading a Stemcell](/bosh.html#upload-a-stemcell).  It should be noted that most kits ship pre-compiled BOSH releases and do not allow the major version of OS of the stemcell to be changed.  If you are currently using `3468.22` you should upload a newer version in the `3468` series, such as `3468.23`.
```